async-nats/src/jetstream/context.rs

```diff
@@ -49,4 +54,25 @@ impl Context {
 
         Ok(response)
     }
+
+    pub async fn add_stream<S>(&mut self, stream_config: S) -> Result<StreamInfo, Error>
```

Comment:
Add versus create? endpoint is create.

```suggestion
    pub async fn create_stream<S>(&mut self, stream_config: S) -> Result<StreamInfo, Error>
```

Reply:
and all APIs are add.... hm.